src/schematools/events/partial.py

```diff
@@ -0,0 +1,549 @@
+__doc__ = """
+
+Direct de tabel bijwerken, of via een tussenstap (met validaties)
+Validaties wschl. eerder in het proces (bij de ingest)
+
+"""
+import json
+import logging
+from collections import defaultdict
+from dataclasses import dataclass
+from typing import Callable, Dict, List, Optional
+
+from sqlalchemy import Column, ForeignKey, Integer, MetaData, String, Table
+
+from schematools import MAX_TABLE_NAME_LENGTH
+from schematools.importer import fetch_col_type, get_table_name
+from schematools.types import DatasetSchema
+from schematools.utils import to_snake_case
+
+metadata = MetaData()
+
+# Enable the sqlalchemy logger to debug SQL related issues
+# logging.basicConfig()
+# logging.getLogger("sqlalchemy.engine").setLevel(logging.INFO)
+
+logger = logging.getLogger(__name__)
+
+
+class UnknownRelationException(Exception):
+    """Custom Exception to signal the fact that an incoming
+    event is unsing an unknown type of relation for
+    which we do not have configuration info available.
+    """
+
+    pass
+
+
+def fetch_insert_data(event_data):
```

Comment:
ik zie veel "fetch_" die niets ophalen, maar alleen bestaande data uitzoeken. Zoiets zou ik altijd "get" noemen. fetch is bijvoorbeeld een HTTP resource ophalen.